.gitignore

```diff
@@ -1,5 +1,6 @@
 node_modules
 .DS_Store
+.idea
```

Comment:
I mentioned this to @bijujoseph too, but might be more convenient for you to add this to your global gitignore so you don't need to add it to every repo.